config/locales/modules/alpha.yml

```diff
@@ -12,6 +12,27 @@ en:
 
           %Warning: people like stuff!%
 
+          <div class="govuk-inset-text prompt prompt">
+            <h2 class="govuk-heading-m">Practitioner prompt</h2>
```

Comment:
Can we replace the content with something useful to developers or the content team, rather than genuine course content.